code updated according style guide

verae98 opened the pull request:
This code was not up to date with the style guide. This needs to be updated on release

verae98 committed:
code updated according style guide

verae98 committed:
Merge branch 'release' into feature-motor_control_moving_platform

b9a9s9 commented on code/headers/motor_control.hpp:
```diff
@@ -8,23 +8,23 @@ namespace r2d2::moving_platform {
+        /**
+		    * the speed set must not exceed 100
+		    */
+        virtual void set_speed(int8_t new_speed) = 0;
```
```suggestion
        virtual void set_speed(int8_t &new_speed) = 0;
```
new_speed can be passed by reference

verae98 replied:
It is indeed necessary to use const & to compile. with ints this is slower

verae98 committed:
Update code/headers/motor_control.hpp

That is an error travis gives indeed. But int normally has no const &, since it is slower

Co-Authored-By: Bas <31634060+b9a9s9@users.noreply.github.com>

verae98 commented on code/headers/motor_control.hpp:
```diff
@@ -8,24 +8,24 @@ namespace r2d2::moving_platform {
+        /**
+		    * the speed set must not exceed 100
+		    */
+        virtual void set_speed(int8_t &new_speed) = 0;
```
```suggestion
        virtual void set_speed(const int8_t &new_speed) = 0;
```

verae98 committed:
Update code/headers/motor_control.hpp
